internal/db/storage/blob/s3/option.go

```diff
@@ -63,6 +65,26 @@ func WithBucket(bucket string) Option {
 	}
 }
 
+// WithReader returns the option to set the reader.
+func WithReader(r reader.Reader) Option {
+	return func(c *client) error {
+		if r != nil {
+			c.reader = r
+		}
+		return nil
+	}
+}
+
+// WithWriter returns the option to set the reader.
+func WithWriter(w writer.Writer) Option {
+	return func(c *client) error {
+		if w != nil {
+			c.writer = w
+		}
+		return nil
+	}
+}
+
```

Comment:
I could not find the test about those functional options.

Reply:
sorry. I forgot to add test.
I will fix it soon